php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -734,6 +738,44 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
         }
     }
 
+    /**
+     * Parses the metadata of a LINST instrument and sets the proper variables
```

Comment:
Why make this support LINST specific instead of putting it in the base class? I can't think of any reason you wouldn't want to allow other instruments to set this metadata and/or would want a different format for the meta file based on the type of instrument..

Reply:
wait... I'm not sure if your saying to apply the same logic for PHP instruments?

its not the same logic, ie PHP instruments already set the table and testnames variables in their setup functions usually (or at least in all the instruments and templates we currently have).

However I just looked at the similar function `loadInstrumentRules` and its indeed not specific to Linst... so I guess i will just modify the documentation to not specifically mention LINST although i find it improbable that PHP instruments would use it

Reply:
@driusan i did make the changes in the instrument class not this one since this is the LINST class the doc seems appropriate